### PR TITLE
OSDOCS-3030: Added RN for AWS Marketplace

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
CP to 4.11

Issue:
This PR is the RN for [OSDOCS-3030](https://issues.redhat.com/browse/OSDOCS-3030)

Link to docs preview:
[AWS Marketplace offering](http://file.rdu.redhat.com/mpytlak/MarketplaceRN/release_notes/ocp-4-11-release-notes.html#ocp-4-11-aws-marketplace)

### CODE REVIEW

**Comment by mjpytlak** [comment]
@dmesser PTAL. IIRC you said the offering was available in North America and EMEA,but your presentation for 4.11 only states NA. EMEA is listed as available end of August.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -179,6 +179,12 @@ For more information, see the "Installation configuration parameters" section of
 The documentation for disabling the Bare Metal Operator is currently unavailable.
 ====
 
+[id="ocp-4-11-aws-marketplace"]
+==== AWS Marketplace offering
+{product-title} is now available on the AWS Marketplace.The AWS Marketplace offering is available to customers who procure {product-title} in North America.
+
+For more information, see link:https://access.redhat.com/articles/6675791[Installing OpenShift using AWS Marketplace].
+
```
Comment (dmesser) [inline release_notes/ocp-4-11-release-notes.adoc:187]: EMEA will follow shortly. Otherwise /lgtm

Comment (mjpytlak) [reply release_notes/ocp-4-11-release-notes.adoc:187]: Thanks @dmesser. I cannot add a future statement to the doc. However, as part of the work I will be doing in the help, I propose a section that details where the AWS (and Azure) marketplace offerings are available. This section can be updated as availability grows. WDYT?

Comment (mjpytlak) [reply release_notes/ocp-4-11-release-notes.adoc:187]: > I cannot add a future statement to the doc. However, as part of the work I will be doing in the help, I propose a section that details where the AWS (and Azure) marketplace offerings are available. This section can be updated as availability grows. WDYT?

Confirmed with @dmesser in Slack that this approach is OK. PR is approved.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -185,6 +185,12 @@ The documentation for disabling the Bare Metal Operator is currently unavailable
 
 For more information, see link:https://access.redhat.com/articles/6484031[Installing OpenShift using Azure Marketplace].
 
+[id="ocp-4-11-aws-marketplace"]
+==== AWS Marketplace offering
+{product-title} is now available on the AWS Marketplace.The AWS Marketplace offering is available to customers who procure {product-title} in North America.
```
Comment (jldohmann) [inline release_notes/ocp-4-11-release-notes.adoc:190]:
```suggestion
{product-title} is now available on the AWS Marketplace. The AWS Marketplace offering is available to customers who procure {product-title} in North America.
```

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -1212,7 +1218,12 @@ With this update, the CMO now waits until deployments are deleted before recreat
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069068[*BZ#2069068*])
 
 * Before this update, for user workload monitoring, if you configured external labels for metrics in Prometheus, the CMO did not correctly propagate these labels to Thanos Ruler.
+<<<<<<< HEAD
```
Comment (jldohmann) [inline release_notes/ocp-4-11-release-notes.adoc:1221]: Merge conflict artifact 🙂

File: release_notes/ocp-4-11-release-notes.adoc [hunk 4]
```diff
@@ -1212,7 +1218,12 @@ With this update, the CMO now waits until deployments are deleted before recreat
...
 If you queried external metrics for user-defined projects, not provided by the user workload monitoring instance of Prometheus, you would sometimes not see external labels for these metrics even though you had configured Prometheus to add them. With this update, the CMO now properly propagates the external labels that you configured in Prometheus to Thanos Ruler, and you can see the labels when you query external metrics.
+=======
```
Comment (jldohmann) [inline release_notes/ocp-4-11-release-notes.adoc:1223]: I'm guessing also a merge conflict artifact

File: release_notes/ocp-4-11-release-notes.adoc [hunk 5]
```diff
@@ -1212,7 +1218,12 @@ With this update, the CMO now waits until deployments are deleted before recreat
...
+Thus, for user-defined projects, if you queried external metrics not provided by the user workload monitoring instance of Prometheus, you would sometimes not see external labels for these metrics even though you had configured Prometheus to add them.
+With this update, the CMO now properly propagates the external labels that you configured in Prometheus to Thanos Ruler, and you can see the labels when you query external metrics.
```
Comment (jldohmann) [inline release_notes/ocp-4-11-release-notes.adoc:1225]: This is a hard return, can it be combined with the line above it?

File: release_notes/ocp-4-11-release-notes.adoc [hunk 6]
```diff
@@ -1212,7 +1218,12 @@ With this update, the CMO now waits until deployments are deleted before recreat
...
+>>>>>>> c9db2bc648 (OSDOCS-3030: Added RN for AWS and Azure Marketplace)
```
Comment (jldohmann) [inline release_notes/ocp-4-11-release-notes.adoc:1226]: Merge conflict artifact

**Review by jldohmann — COMMENTED** [review]
Mostly looking good, feedback below

File: release_notes/ocp-4-11-release-notes.adoc (hunk 3 above)
Comment (mjpytlak) [reply release_notes/ocp-4-11-release-notes.adoc]: Ooofff. Thanks.

**Review by jldohmann — COMMENTED** [review]
/lgtm

**Pull request merged**